app/src/main/java/com/io/moviO/moviesList/MovieListAdapter.kt

```diff
@@ -23,7 +24,9 @@ class MovieListAdapter(
 
         fun bind(movie: Movie, listener: OnMovieClickedListener) {
             binding.movieNameMovieListTv.text = movie.name
```

Comment:
all binding related calls could be scoped into an `apply` function

Reply:
resolved